### PR TITLE
add url

### PR DESCRIPTION
**Pull request by szymonsso**
Fixes #some issue with download

### CODE REVIEW

File: python_analizator.py [hunk 1]
```diff
@@ -10,7 +10,8 @@
 
 
 def download():
-
+    print('Download Starting...')
```
Comment (naumowicz) [inline python_analizator.py:13]: Please remove this

**Review by KarolinaAntonowicz — APPROVED** [review]

File: 6.txt [hunk 2]
```diff
@@ -0,0 +1 @@
...re in a better frame of mind. Rock music approaches at high velocity. Please wait outside of the house. He said he was not there yesterday; however, many people saw him there. Last Friday in three week’s time I saw a spotted striped blue worm shake hands with a legless lizard. I am never at home on Sundays. Sometimes, all you need to do is completely make an ass of yourself and laugh it off to realise that life isn’t so bad after all. Hurry! I currently have 4 windows open up… and I don’t know why. Mary plays the piano. She advised him to come back at once. Abstraction is often one floor above you. He ran out of money, so he had to stop playing poker. I often see the time 11:11 or 12:34 on clocks. She wrote him a long letter, but he didn't read it. I checked to make sure that he was still alive. Wow, does that work? I want more detailed information. There was no ice cream in the freezer, nor did they have money to go to the store. The sky is clear; the stars are twinkling. It was getting dark, and we weren’t there yet. Should we start class now, or should we wait for everyone to get here? She was too short to see over the fence. I am counting my calories, yet I really want dessert.
```
Comment (naumowicz) [inline 6.txt]: unnecessary file

**Review by naumowicz — CHANGES_REQUESTED** [review]

File: python_analizator.py (hunk 1 above)
Comment (KarolinaAntonowicz) [reply python_analizator.py:13]: I like this

File: 6.txt (hunk 2 above)
Comment (KarolinaAntonowicz) [reply 6.txt]: please reomve this file

**Comment by szymonsso** [comment]
File deleted, should be fine

**Pull request merged**